analyses/molecular-subtyping-HGG/07-HGG-molecular-subtyping-combine-table.Rmd

```diff
@@ -95,7 +95,23 @@ dna_df <- mutation_df %>%
 ```
 
 In earlier work on this module, we observed duplicated `sample_id`.
-We removed non-tumor samples (e.g., derived cell lines), does this issue persist?
+Yes
```

Comment:
```suggestion
```
Just to add to @jaclyn-taroni's comment on this - if you remove, then disregard. But, since you removed the question, I removed the answer.